File: py/ds/queue.py
class Queue:
    def __init__(this):
        self.items = []

    def isEmpty(self):
        return self.items == []

    def enqueu(self, item):
        self.items.insert(0,item)

    def dequeu(self):
        return self.items.pop()

    def size(self):
        return len(self.item)

File: js/ds/queue.js
class Queue {
  constructor () {
    this.items = [];
  }

  isEmpty () {
    return this.items === [];
  }

  enqueue (item = undefined) {
    if (item) {
      this.items.unshift(item);
    }
  }

  dequeue () {
    return this.items.pop();
  }

  size () {
    return this.items.length;
  }
}
